src/Upgrader/Business/DataProvider/Client/ReleaseApp/Http/Response/HttpResponseInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrader\Business\DataProvider\Client\ReleaseApp\Http\Response;
+
+interface HttpResponseInterface
+{
+    /**
+     * @param int $code
+     * @param string $body
+     */
+    public function __construct(int $code, string $body);
```

Comment:
I can assume __construct method can be avoided in the interface.

Reply:
deleted